src/app/helpers/lazyImageLoading.js

```diff
@@ -0,0 +1,33 @@
+const createLazyListener = (selector, dataset) => () => {
+  let images = document.querySelectorAll(selector);
+  if (images.length) {
+    images = [].slice.call(images);
+    setTimeout(() => {
```

Comment:
why do you need a `setTimeout` here?
and the second one for deleting a dataset? is it so bad to keep it there?

Reply:
The second one is just for view beauty
First one was in this func, it worked with bugs before my changes, ten I just forgot to remove it:(

Reply:
> why do you need a `setTimeout` here?
> and the second one for deleting a dataset? is it so bad to keep it there?

The second one is just for view beauty
First one was in this func, it worked with bugs before my changes, ten I just forgot to remove it:(

Reply:
> why do you need a `setTimeout` here?
> and the second one for deleting a dataset? is it so bad to keep it there?

First one was in this func, it worked with bugs before my changes, ten I just forgot to remove it:(
They had a purpose, but again, function changed, sorry for that :(

Reply:
Second one is for delay before remove blur styles from image